### PR TITLE
Adding alloy to chemical mixture type

### PR DESCRIPTION
**Pull request by Peyman-N**

### CODE REVIEW

File: instances/chemicalMixtureType/alloy.jsonld [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/chemicalMixtureType/alloy",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/ChemicalMixtureType",
+  "definition": "A mixture consisting of a combination of metals or of a metal and another element that retains the characteristics of a metal.",
+  "description": "",
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "Alloy",
```
Comment (lzehl) [inline instances/chemicalMixtureType/alloy.jsonld:11]: please do not capitalize

File: instances/chemicalMixtureType/alloy.jsonld [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/chemicalMixtureType/alloy",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/ChemicalMixtureType",
+  "definition": "A mixture consisting of a combination of metals or of a metal and another element that retains the characteristics of a metal.",
```
Comment (lzehl) [inline instances/chemicalMixtureType/alloy.jsonld:7]: Please always add your source that you used for the definition. 

We should discuss if we either always want to start with the name of the entity we are defining or not. I would like to harmonize this a bit.

File: instances/chemicalMixtureType/alloy.jsonld [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "description": "",
```
Comment (lzehl) [inline instances/chemicalMixtureType/alloy.jsonld:8]: this should be null if it is not provided

**Review by lzehl — CHANGES_REQUESTED** [review]

**Comment by UlrikeS91** [comment]
replaced by https://github.com/openMetadataInitiative/openMINDS_instances/pull/86

**Pull request closed without merging**